content/client-server-api/modules/rich_replies.md

```diff
@@ -18,9 +19,22 @@ Additionally, a rich reply can reference any other event type as of v1.3.
 Previously, a rich reply could only reference another `m.room.message` event.
 {{% /boxes/note %}}
 
-When possible, events SHOULD include a [fallback representation](#fallbacks-for-rich-replies)
-to allow clients which do not render rich replies to still see something which
-appears to be a quoted reply.
+{{% boxes/note %}}
+{{% changed-in v="1.13" %}}
+In previous versions of the specification, rich replies could include a fallback
+format in the `body` and `formatted_body` for clients that do not support rich
+replies, this is no longer the case. Clients might still want to remove this
+fallback before rendering the event.
```

Comment:
I think they *must* remove it, no?

Reply:
There is not _must_ in the MSC. There is:

> Clients […] should consider treating `<mx-reply>` parts as they treat other invalid html tags.

And

> […]  clients may need to strip out such fallbacks.

Reply:
My understanding is that it should not be mandatory as it will allow clients to drop any special code regarding this feature when it is not sent anymore in new messages. It will only appear in historical events but it could be considered acceptable to display the fallback in that case.